tests/testthat/_snaps/tmb-methods.md

```diff
@@ -5,6 +5,7 @@
     Formula:     FEV1 ~ RACE + us(AVISIT | USUBJID)
     Data:        fev_data (used 537 observations from 197 subjects with maximum 4 
     timepoints)
+    Weights:     rep(1, nrow(fev_data))
```

Comment:
hm this is not very nice. I think therefore I had some additional logic before to circumvent this from being printed

Reply:
I think previously only when `weights` is assigned (not rep(1, nrow(data)) then it is displayed. do you mean this https://github.com/openpharma/mmrm/blob/b3f7e5c7d53ee605e4b2041aba48a3f9c42b7878/R/fit.R#L369

so in previous logic, weights get displayed when:

in `mmrm`, weights argument is used, not NULL; in `fit_mmrm`, no matter if weights is assigned, it is not displayed

here the change is that in `fit_mmrm` the assigned `weights` still get displayed. we can of course remove that "weights" for `fit_mmrm$call`. but why we should do that? in `mmrm` by default the weights are not displayed

Reply:
ah ok cool - so when we use `mmrm()` without explicit weights assigned it is not displayed still right? then it is ok.

Reply:
it only gets printed when you have "weights" argument specified; e.g. using `mmrm(formula, data = fev_data)` you don't have the weights printed. If you have `mmrm(weights = xxx)` it got printed (this is the same for previous version)

Reply:
(and that's why `fit_mmrm` and `mmrm` both keep a `call` but they always use the call they received)